pkg/controllers/management/node/controller.go

```diff
@@ -44,8 +45,11 @@ import (
 	typedv1 "k8s.io/client-go/kubernetes/typed/core/v1"
 )
 
+var (
+	defaultEngineInstallURL = settings.EngineInstallURL.Get()
```

Comment:
This is just used once in the module so better option would be to use `settings.EngineInstallURL.Get()` here directly https://github.com/rancher/rancher/blob/61010cc507177fa55128b5443c1180d0c69d5c80/pkg/controllers/management/node/controller.go#L209

Reply:
Thank you, updated.